Src/library/base/elks/kernel/ise_runtime.e

```diff
@@ -812,9 +762,7 @@ feature -- Object marking
 			-- Get a lock on `mark' and `unmark' routine so that 2 threads cannot `mark' and
 			-- `unmark' at the same time.
 		external
-			"C blocking use %"eif_traverse.h%""
-		alias
-			"eif_lock_marking"
+			"built_in static"
```

Comment:
How do you handle the "blocking" semantic?

Reply:
Good question, I will move this code to the builtin class.

Reply:
Done